src/torrent.ts

```diff
@@ -168,6 +168,18 @@ export async function getInfoHashesToExclude(): Promise<string[]> {
 	);
 }
 
+export async function validateAction(): Promise<void> {
```

Comment:
There might be a better home for this file—maybe `action.ts` or somewhere else we have a lot of validation code?

Reply:
Moved to action.ts